src/common/regions.ts

```diff
@@ -119,16 +141,62 @@ const counties: County[] = chain(state_county_map_dataset)
   })
   .value();
 
+const countiesByFips = fromPairs(
+  counties.map(county => [county.fipsCode, county]),
+);
+
+// Seattle-Tacoma, WA -> seattle-tacoma_wa
+function urlFormat(name: string) {
+  return name
+    .toLowerCase()
+    .split(', ')
+    .map(part => deburr(words(part).join('-')))
+    .join('_');
+}
+
+const metroAreas: MetroArea[] = msaDefinitions.msa_regions.map(metroArea => {
+  // TODO: Handle counties with FIPS: 36005, 360047, 36081, 36085, those
+  // are part of MSAs, but not in the countiesByFips map
+  const metroAreaCounties: County[] = metroArea.countyFips
+    .map(countyFips => countiesByFips[countyFips])
+    .filter(county => county);
```

Comment:
TODO: Investigate why we don't have those counties in our list of counties and either add them or remove them from the metro areas file (via the script)

Reply:
I think those are the New York City boroughs that we aggregate into a single fake county thing.  Tom and I are working on unraveling that this week.  I'll probably send you a PR tonight or tomorrow.